### DISCUSSION TITLE
[Feat] 支持偏好设置中设置 输入预处理 / 系统预设注入

### DISCUSSION THREAD

**Discussion by yh4922** [coaidev/coai#287] [Discussions | 交流]
希望能支持 **ChatGPT-Next-Web** 的预设对话和输入预处理的功能
能支持的话 对话的可定制性会高很多
![image](https://github.com/Deeptrain-Community/chatnio/assets/18353610/a4cf84fd-3f63-4176-9878-bd7e96755d0c)

**Comment by Hirx0x01** [comment]
好建议

**Comment by zmh-program** [comment]
是我很久之前就想写的功能了，等我忙完这段时间后增加此项功能。

**Comment by bmwa0813** [comment]
还有预设pinned功能，商业版很需要呀😘

[a comment by an automated account is not shown]
